### PR TITLE
added deployment scripts for sista VM

### PR DESCRIPTION
**Pull request by clementbera**
I don't know how to check it works. Can some one review this ? Esteban or Fabio ?
Else I will merge next week and check if nothing breaks.

### CODE REVIEW

**Comment by fniephaus** [comment]
Does the `../vm/sista` directory exist on the server? Maybe it needs to be created manually before your changes work. AFAICT, the changes look good so far but it's better @estebanlm has a look as well and merges this.

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ jobs:
   - env: ARCH="linux32x86" FLAVOR="pharo.cog.spur" HEARTBEAT="threaded"
   - env: ARCH="linux32x86" FLAVOR="pharo.cog.spur" HEARTBEAT="itimer"
   - env: ARCH="linux32x86" FLAVOR="pharo.sista.spur" HEARTBEAT="threaded"
+  - env: ARCH="linux32x86" FLAVOR="pharo.sista.spur" HEARTBEAT="itimer"
```
Comment (fniephaus) [inline .travis.yml:34]: why is the itimer version needed?

File: deploy/pharo/filter-exec.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ readonly PR_SHA="${TRAVIS_PULL_REQUEST_SHA:-${APPVEYOR_PULL_REQUEST_HEAD_COMMIT}
 readonly BRANCH_NAME="${TRAVIS_BRANCH:-${APPVEYOR_REPO_BRANCH}}"
 readonly TAG_NAME="${TRAVIS_TAG:-${APPVEYOR_REPO_TAG_NAME}}"
 
-if [[ "${FLAVOR}" != "pharo.cog.spur" ]]; then
+if ! ([[ "${FLAVOR}" = "pharo.cog.spur" ]] || [[ "${FLAVOR}" = "pharo.sista.spur" ]]); then
```
Comment (fniephaus) [inline deploy/pharo/filter-exec.sh:15]: Line 5 needs to be updated as well ;)

**Review by fniephaus — APPROVED** [review]

**Comment by clementbera** [comment]
Let's just merge and we will see if it works.

**Pull request merged**